Spring/KeyboardLayoutConstraint.swift

```diff
@@ -23,88 +23,87 @@
-    
     @available(tvOS, unavailable)
-    override public func awakeFromNib() {
-        super.awakeFromNib()
-        
-        offset = constant
-        
-        NotificationCenter.default.addObserver(self, selector: #selector(KeyboardLayoutConstraint.keyboardWillShowNotification(_:)), name: UIWindow.keyboardWillShowNotification, object: nil)
-        NotificationCenter.default.addObserver(self, selector: #selector(KeyboardLayoutConstraint.keyboardWillHideNotification(_:)), name: UIWindow.keyboardWillHideNotification, object: nil)
-    }
-    
-    deinit {
-        NotificationCenter.default.removeObserver(self)
-    }
-    
-    // MARK: Notification
-    
-    @objc func keyboardWillShowNotification(_ notification: Notification) {
-        if let userInfo = notification.userInfo {
-            if let frameValue = userInfo[UIResponder.keyboardFrameEndUserInfoKey] as? NSValue {
-                let frame = frameValue.cgRectValue
-                keyboardVisibleHeight = frame.size.height
-            }
-            
-            self.updateConstant()
-            switch (userInfo[UIResponder.keyboardAnimationDurationUserInfoKey] as? NSNumber, userInfo[UIResponder.keyboardAnimationCurveUserInfoKey] as? NSNumber) {
-            case let (.some(duration), .some(curve)):
-                
-                let options = UIView.AnimationOptions(rawValue: curve.uintValue)
-                
-                UIView.animate(
-                    withDuration: TimeInterval(duration.doubleValue),
-                    delay: 0,
-                    options: options,
-                    animations: {
-                        UIApplication.shared.keyWindow?.layoutIfNeeded()
-                        return
-                    }, completion: { finished in
-                })
-            default:
-                
-                break
+    public class KeyboardLayoutConstraint: NSLayoutConstraint {
+
+        private var offset: CGFloat = 0
+        private var keyboardVisibleHeight: CGFloat = 0
+
+        @available(tvOS, unavailable)
+        public override func awakeFromNib() {
+            super.awakeFromNib()
+
+            offset = constant
+
+            NotificationCenter.default.addObserver(self, selector: #selector(KeyboardLayoutConstraint.keyboardWillShowNotification(_:)), name: UIWindow.keyboardWillShowNotification, object: nil)
+            NotificationCenter.default.addObserver(self, selector: #selector(KeyboardLayoutConstraint.keyboardWillHideNotification(_:)), name: UIWindow.keyboardWillHideNotification, object: nil)
+        }
+
+        deinit {
+            NotificationCenter.default.removeObserver(self)
+        }
+
+        // MARK: Notification
+
+        @objc func keyboardWillShowNotification(_ notification: Notification) {
+            if let userInfo = notification.userInfo {
+                if let frameValue = userInfo[UIResponder.keyboardFrameEndUserInfoKey] as? NSValue {
```

Comment:
'keyboardFrameEndUserInfoKey' has been renamed to 'UIKeyboardFrameEndUserInfoKey'

Reply:
Which target are you building when this warning appears ?
I just build again and I don't have anything.. 🤔